### PR TITLE
Added metaeuk recipe

### PR DESCRIPTION
**Pull request by milot-mirdita**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [*] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [*] This PR adds a new recipe.
* [*] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dpryan79** [comment]
We really try to not include prereleases, can you tag a non-prerelease?

File: recipes/metaeuk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set version = "0-c87cf06" %}
```
Comment (dpryan79) [inline recipes/metaeuk/meta.yaml]: When you do tag a regular release, can you use regular versioning, such as 0.1.0?

**Comment by milot-mirdita** [comment]
Thanks for the review and sorry for the late reply. We still had to fix a bug when trying to run with very little input.

We generally do not use semantic versioning for our various methods. The versioning scheme is `incrementingNumber-commitHash`, since we do not have the bandwidth to support previous releases. We try our best to keep the CLI stable though.

**Comment by dpryan79** [comment]
Almost no one supports previous releases, semantic versioning primarily aids users in figuring out how drastic a change in version is.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**